lib/tasks/braintree_tokens.rake

```diff
@@ -0,0 +1,8 @@
+namespace :champaign do
+  desc "Create braintree payment method tokens out of existing token columns"
+  task braintree_tokens: :environment do
+    Payment::BraintreeCustomer.all.each do |customer|
+      Payment::BraintreePaymentMethod.create!({customer_id: customer.customer_id, token: customer.card_vault_token})
```

Comment:
Yes, absolutely. It's actually odd that the schema is out of date since I've been dropping and recreating the DB left and right :o It should be up to date now.